<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Mypc" name="com.stockAccounting.Tests.grid" tests="1" failures="0" timestamp="4 Aug 2016 05:57:25 GMT" time="1.553" errors="1">
  <testcase name="gt" time="1.553" classname="com.stockAccounting.Tests.grid">
    <error type="org.openqa.selenium.WebDriverException" message="Error forwarding the new session cannot find : Capabilities [{platform=WINDOWS, ensureCleanSession=true, browserName=internet explorer, version=}]
Command duration or timeout: 407 milliseconds
Build info: version: &#039;2.53.0&#039;, revision: &#039;35ae25b&#039;, time: &#039;2016-03-15 17:00:58&#039;
System info: host: &#039;Mypc&#039;, ip: &#039;192.168.1.24&#039;, os.name: &#039;Windows 8.1&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.3&#039;, java.version: &#039;1.7.0_79&#039;
Driver info: org.openqa.selenium.remote.RemoteWebDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: Error forwarding the new session cannot find : Capabilities [{platform=WINDOWS, ensureCleanSession=true, browserName=internet explorer, version=}]
Command duration or timeout: 407 milliseconds
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 17:00:58'
System info: host: 'Mypc', ip: '192.168.1.24', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.7.0_79'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:249)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:158)
	at com.stockAccounting.Tests.grid.gt(grid.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
Caused by: org.openqa.grid.common.exception.GridException: Error forwarding the new session cannot find : Capabilities [{platform=WINDOWS, ensureCleanSession=true, browserName=internet explorer, version=}]
	at org.openqa.grid.web.servlet.handler.RequestHandler.process(RequestHandler.java:115)
	at org.openqa.grid.web.servlet.DriverServlet.process(DriverServlet.java:83)
	at org.openqa.grid.web.servlet.DriverServlet.doPost(DriverServlet.java:67)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.seleniumhq.jetty9.servlet.ServletHolder.handle(ServletHolder.java:808)
	at org.seleniumhq.jetty9.servlet.ServletHandler.doHandle(ServletHandler.java:587)
	at org.seleniumhq.jetty9.server.session.SessionHandler.doHandle(SessionHandler.java:221)
	at org.seleniumhq.jetty9.server.handler.ContextHandler.doHandle(ContextHandler.java:1127)
	at org.seleniumhq.jetty9.servlet.ServletHandler.doScope(ServletHandler.java:515)
	at org.seleniumhq.jetty9.server.session.SessionHandler.doScope(SessionHandler.java:185)
	at org.seleniumhq.jetty9.server.handler.ContextHandler.doScope(ContextHandler.java:1061)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.seleniumhq.jetty9.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)
	at org.seleniumhq.jetty9.server.Server.handle(Server.java:499)
	at org.seleniumhq.jetty9.server.HttpChannel.handle(HttpChannel.java:310)
	at org.seleniumhq.jetty9.server.HttpConnection.onFillable(HttpConnection.java:257)
	at org.seleniumhq.jetty9.io.AbstractConnection$2.run(AbstractConnection.java:540)
	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- gt -->
</testsuite> <!-- com.stockAccounting.Tests.grid -->
